Bottom: fb2e0df0be78f8508eeaa2e8bd0541bce8bff627
Top:    22cdce30b3f34eaf3fb19fa68e5f8b17662531a1
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-11-09 15:30:15 +0800

Refresh of tuntap-rx-batching

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index ea59dd4..d40583b 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -636,7 +636,7 @@ static int tun_poll(struct napi_struct *napi, int budget)
 	struct sk_buff_head *input_queue =
 	       &tfile->socket.sk->sk_write_queue;
 	struct sk_buff *skb;
-	unsigned received = 0;
+	unsigned int received = 0;
 
 	while (1) {
 		while ((skb = __skb_dequeue(&tfile->process_queue))) {
